### ISSUE TITLE
Make JSON version of electorate pages

### ISSUE THREAD

**Issue by henare** [pat/my_candidates#10]
When you change the filetype of an electorate page from `html` to `json` it should show you the Popolo data for that page, e.g. http://votingchoices.com/electorates/barton.json

**Comment by pat** [comment]
Yes, that would be neat :) Not sure I've got the popolo data formatted quite correctly, though… so I guess the first step is to verify that.

**Comment by henare** [comment]
This is a useful gem for manipulating Popolo that we've been contributing to recently: https://github.com/everypolitician/everypolitician-popolo
